pyfr/progress_bar.py

```diff
@@ -1,20 +1,25 @@
+import contextlib
+import itertools as it
 import shutil
 import sys
 import time
 
 
-def to_hms(delta):
-    hours, remainder = divmod(int(delta), 3600)
-    minutes, seconds = divmod(remainder, 60)
+def format_hms(delta):
```

Comment:
Might it be a bit simpler to use the built-in datetime package?

Reply:
The equivalent class is `datetime.timedelta` but you are on your own when it comes to formatting.